libr/io/p/io_debug.c

```diff
@@ -167,7 +167,7 @@ static int fork_and_ptraceme(RIO *io, int bits, const char *cmd) {
 	if (th != INVALID_HANDLE_VALUE) CloseHandle (th);
 
 	eprintf ("Spawned new process with pid %d, tid = %d\n", pid, tid);
-	io->winbase = de.u.CreateProcessInfo.lpBaseOfImage;
+	io->winbase = (HANDLE)(ut64)de.u.CreateProcessInfo.lpBaseOfImage;
```

Comment:
this is wrong. you cant cast an immediate with to a pointer without a size_t cast